pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -280,7 +278,9 @@ func (r *ReconcileComplianceSuite) reconcileScanRemediations(suite *complianceop
 			// All remediation lists in the scan must be equal
 			ok := diffRemediationList(scanRemediations, resultRemediations)
 			if !ok {
-				return fmt.Errorf("the list of remediations differs")
+				// TODO(jaosorior): Update suite status to reflect that there's an issue
+				// either with MCO or a compromise on a host.
```

Comment:
Should we file a Jira or an Issue? (Yes, I am very guilty of just putting FIXMEs and TODOs in the code, too)

Reply:
Let's open a jira

Reply:
I amended the acceptance criteria of 306